PhotoStream/app/src/main/java/com/keenanzucker/photostream/ImageCallback.java

```diff
@@ -0,0 +1,13 @@
+package com.keenanzucker.photostream;
+
+import java.util.ArrayList;
+
+/**
+ * Created by keenan on 9/17/15.
+ * Interface for image callback. Returns a arraylist of image urls
+ */
+public interface ImageCallback {
+
+    void callback(ArrayList<String> imageSRCs);
```

Comment:
Like I said above, it may make more sense for this to just take in a JsonObject and then also do all the processing that lines 53-63 above does